portal/views/patients.py

```diff
@@ -136,12 +136,16 @@ def get_orgs_consent_agreements():
     consent_agreements = {}
     for org_id in OrgTree().all_top_level_ids():
         org = Organization.query.get(org_id)
-        asset, url = VersionedResource.fetch_elements(
+        dict_consent_by_org = VersionedResource.fetch_elements(
             app_text(ConsentByOrg_ATMA.name_key(organization=org)))
+        asset = dict_consent_by_org['asset'] if 'asset' in dict_consent_by_org else None
```

Comment:
Would you be able to change these to use the [`get()` method of dictionaries](https://pythontips.com/2013/07/29/dictionaries-have-a-get-method/) (see below)? I know it's a lot of replacements (sorry!) but it's a little easier to read.

Thanks!


```
asset = dict_consent_by_org.get('asset', None)
```